### PR TITLE
move the buildpackages task before the install task

### PR DESCRIPTION
**Pull request by dachary**
http://tracker.ceph.com/issues/13031

### CODE REVIEW

File: teuthology/task/internal.py [hunk 1]
```diff
@@ -342,6 +342,50 @@ def push():
     except Exception:
         log.exception("Error pushing inventory")
 
+BUILDPACKAGES_SWAPPED = 0
+BUILDPACKAGES_OK = 1
+BUILDPACKAGES_REMOVED = 2
+BUILDPACKAGES_NOTHING = 3
+
+def buildpackages(ctx, config):
```
Comment (zmc) [inline teuthology/task/internal.py]: Curious... why isn't this implemented as a class-based ``Task`` with https://github.com/ceph/teuthology/blob/master/teuthology/task/__init__.py#L9 ?

Comment (dachary) [inline teuthology/task/internal.py]: I'll use that class template, thanks ! This is not a task though, just a function shuffling tasks.

**Comment by dachary** [comment]
@zmc ping ?

**Comment by zmc** [comment]
@dachary this looks like a duplicate of #623?

**Comment by dachary** [comment]
@zmc my bad indeed

**Pull request closed without merging**